force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +97,22 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<String> words = sentence.split(' ');
 
         // Create a set to store the unique words
+        Set<String> uniqueWords = new Set<String>();
 
         // Add each word to the set
+        for (String word : words) {
+            uniqueWords.add(word);
+        }
+        
```

Comment:
Can you avoid looping here by just adding all of the values at once using a method on the set?